src/components/attendee/Header/AttendeeHeader.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useState } from "react";
```

Comment:
There is a file under `src/components/attendee/AttendeeHeader/AttendeeHeader.tsx` where I think this one was moved to. Probably should reconcile them first

Reply:
removed this one [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)